src/libcprover-cpp/api.cpp

```diff
@@ -133,57 +133,7 @@ void api_sessiont::load_model_from_files(
     files, *implementation->message_handler, *implementation->options));
 }
 
-void api_sessiont::verify_model() const
```

Comment:
Why has this file being touched by this PR?
It shouldn't be changed.

Reply:
removed, sorry for cluttering